zerver/models.py

```diff
@@ -546,6 +546,7 @@ class RealmDomain(models.Model):
     # should always be stored lowercase
     domain = models.CharField(max_length=80, db_index=True)  # type: str
     allow_subdomains = models.BooleanField(default=False)
+    invite_required = models.BooleanField(default=True)
```

Comment:
So, one problem with this model is it'll break everything when we apply this migration.

My proposal had been to make this a `BooleanField(null=True, default=None)`, with the semantics being that if it is unspecified/Null, we inherit the organization-level setting.  This makes it possible to make changes to the organization-level setting and have that adjust the setting for every domain where the user hasn't explicitly overridden that setting in the past.

The other way we could do this is to include a migration that changes the initial values for existing realms to match the previous model.